addons/knowledge/data/knowledge_data.xml

```diff
@@ -530,6 +532,8 @@
             </field>
             <field name="parent_id" eval="ref('knowledge_article_sales_and_marketing')" />
             <field name="internal_permission">write</field>
+            <field name="article_member_ids" eval="[(5, 0), (0, 0, {'partner_id': ref('base.main_partner'), 'permission': 'read'}),
```

Comment:
Why ? Not sure why we should give member rights on default data... Would keep it as simple as possible, keeping the write internal permission seems sufficient to me.

If you want demo data, should should be done in a demo file.